Submit Assienment/Swapnil_Patil_039/script.js

```
@@ -0,0 +1,161 @@
+      const title = document.createElement('h1');
+      title.classList.add(config.titleClass);
+      title.textContent = (data.title);
+      purple.appendChild(title); 
+
+      const headlineLine = document.createElement('p');
+      headlineLine.classList.add(config.headlineLineClass);
+      headlineLine.textContent = (data.headlineLine);
+      purple.appendChild(headlineLine);
+
+      var hr = document.createElement('hr');
+      hr.classList.add(config.hrClass);
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)